.env.example

```diff
@@ -1,6 +1,6 @@
 # Assumes the Postgres server was started using the Compose file in docker/postgres/docker-compose.yml
 DATABASE_URL=postgres://postgres:@localhost:5432/compass
-
+# generated using `openssl rand -base64 32`
```

Comment:
I think this is already specified below?

Reply:
oops didnt realize I edited this. my bad